demo/scenario/script

```diff
@@ -17,10 +17,37 @@ submit Auctioneer1 createAuction
 
 # clock-rounds ...
 submit Auctioneer1 startNextRound
-wait "enter A-Telecom bid manually ...  (Hit any key to continue)"
+submit A-Telecom submitClockBid
 submit B-Net submitClockBid
+submit C-Mobile submitClockBid
 submit Auctioneer1 endRound
 
+wait "End Round 1  (Hit any key to continue)"
+
+submit Auctioneer1 startNextRound
+submit A-Telecom submitClockBid
+submit B-Net submitClockBid
+submit C-Mobile submitClockBid
+submit Auctioneer1 endRound
+
+wait "End Round 2  (Hit any key to continue)"
+
+submit Auctioneer1 startNextRound
+wait "enter C-Mobile bid manually ...  (Hit any key to continue)"
+submit A-Telecom submitClockBid
+submit B-Net submitClockBid
+submit Auctioneer1 endRound
+
+wait "End Round 3  (Hit any key to continue)"
+
+submit Auctioneer1 startNextRound
+submit A-Telecom submitClockBid
+submit B-Net submitClockBid
+submit C-Mobile submitClockBid
+submit Auctioneer1 endRound
+
+wait "End Round 4 - Clock Phase Complete  (Hit any key to continue)"
+
 # assignment rounds ...
 submit Auctioneer1 startNextRound
 # bidding not yet implemented, add here once done ....
```

Comment:
This is not a problem but note that: the end round here fails because the assignment phase is started **and terminated**  already in `startNextRound`.